src/netbase.h

```diff
@@ -17,6 +17,10 @@
 #include <string>
 #include <vector>
 
+#ifndef WIN32
```

Comment:
It doesn't seem like the current code is using `poll` on Windows.

Reply:
it isnt?

Reply:
```c++
#if defined(__linux__)
#define USE_POLL
#endif
```

Reply:
Oh, damn, I misread if**n**def.